lib/permutations.rb

```diff
@@ -1,4 +1,31 @@
+# Time complexity: O(n) : loop through the char_arr in a linear fashion
+# Space complexity: O(1) : hash_table creation is limited to 26 characters of the alphabet
 
 def permutations?(string1, string2)
```

Comment:
⚠️  This doesn't work, consider that it returns true for  `permutations?("aa", "bb")`